.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -1,31 +1,151 @@
 name: "\U0001F41E Bug report"
-description: Report an issue
-labels: ['s: pending triage'] # This will automatically assign the 's: pending triage' label
+description: "Something isn't working as expected \U0001F914"
+labels: ['s: pending triage', 'c: bug']
 body:
   - type: markdown
     attributes:
-      value: Thanks for taking the time to fill out this bug report!
+      value: |
+        ## :warning: We use GitHub Issues to track bug reports, feature requests and regressions
+
+        If you are not sure that your issue is a bug, you could:
+
+        - use our [Discord community](https://chat.fakerjs.dev)
+        - open a new [discussion](../discussions) and ask your question there
+
+        **NOTE:** You don't need to answer questions that you know that aren't relevant.
+
+  - type: checkboxes
+    attributes:
+      label: 'Is there an existing issue for this?'
+      description: 'Please search [here](./?q=is%3Aissue) to see if an issue already exists for the bug you encountered'
+      options:
+        - label: 'I have searched the existing issues'
+          required: true
+
+  - type: checkboxes
+    attributes:
+      label: 'Have you checked the documentation?'
+      description: 'Please read our [documentation](https://fakerjs.dev/guide) to see if the bug you encountered is a use error'
+      options:
+        - label: "I have read the documentation for the area I'm reporting the bug in"
+          required: true
+
   - type: textarea
-    id: bug-description
+    validations:
+      required: true
     attributes:
-      label: Describe the bug
-      description: A clear and concise description of what the bug is. If you intend to submit a PR for this issue, tell us in the description. Thanks!
-      placeholder: Bug description
+      label: 'Current behavior'
```

A more like call to action would be
```suggestion
      label: 'Describe the bug'
```